src/NuGet.Clients/NuGet.MSSigning.Extensions/packages.lock.json

```diff
@@ -0,0 +1,115 @@
+{
+  "version": 1,
+  "dependencies": {
+    ".NETFramework,Version=v4.7.2": {
+      "PdbGit": {
+        "type": "Direct",
+        "requested": "[3.0.41, )",
+        "resolved": "3.0.41",
+        "contentHash": "LBeCjKYbWdrZHSNVOp/5VusxjmJg4xkU8Pnz9DgSHGXApPYW3RnsOTvfpDErtnACpRaB4W16lHN91id20D4i8w=="
+      },
+      "Microsoft.Web.Xdt": {
+        "type": "Transitive",
+        "resolved": "2.1.2",
+        "contentHash": "YIaNTOOtGzjF8CkQBOx3aUBWi0dD+g/nOjbpsVslx9G0/uldtpmoKLmyldmKdMv/gHs3qg40rBT3+wvEWoY5gg=="
+      },
+      "Newtonsoft.Json": {
+        "type": "Transitive",
+        "resolved": "9.0.1",
+        "contentHash": "U82mHQSKaIk+lpSVCbWYKNavmNH1i5xrExDEquU1i6I5pV6UMOqRnJRSlKO3cMPfcpp0RgDY+8jUXHdQ4IfXvw=="
+      },
+      "NuGet.Build.Tasks": {
+        "type": "Project",
+        "dependencies": {
+          "NuGet.Commands": "5.0.0-preview3"
+        }
+      },
+      "NuGet.CommandLine": {
+        "type": "Project",
+        "dependencies": {
+          "NuGet.Build.Tasks": "5.0.0-preview3",
```

Comment:
When the sln wide versions change from 5.0.0-preview3 to 5.0.0-preview4, we would need to regenerate the lock files, correct?

Reply:
yes, project version is changed so it'd require to regenerate